src/server/types.ts

```diff
@@ -1,25 +1,28 @@
 import { ComponentType } from "preact";
-import { ConnInfo, ServeInit } from "./deps.ts";
+import { ConnInfo, ServeInit, ServeTlsInit } from "./deps.ts";
 import * as router from "./router.ts";
 import { InnerRenderFunction, RenderContext } from "./render.ts";
 
 // --- APPLICATION CONFIGURATION ---
 
-export type StartOptions = ServeInit & FreshOptions & {
+export type StartOptions = ServeInit & FreshOptions & FreshExperimantal;
+export type StartTlsOptions = FreshOptions & ServeTlsInit & FreshExperimantal;
+
+export interface FreshOptions {
+  render?: RenderFunction;
+  plugins?: Plugin[];
+  staticDir?: string;
+  router?: RouterOptions;
+}
```

Comment:
For the future: Moving this interface around creates noise in the diff which makes reviewing a little harder for the persons having to review the code.